Packs/CarbonBlackEnterpriseEDR/Integrations/CarbonBlackEnterpriseEDR/CarbonBlackEnterpriseEDR.py

```diff
@@ -0,0 +1,1314 @@
+        readable_output=readable_output,
+        raw_response=result
+    )
+    return results
+
+
+def alert_workflow_update_command(client: Client, args: Dict) -> CommandResults:
+
+    alert_id = args.get('alert_id')
+    state = args.get('state')
+    comment = args.get('comment')
+    remediation_state = args.get('remediation_state')
+
+    result = client.alert_workflow_update_request(alert_id, state, comment, remediation_state)
+
+    readable_output = tableToMarkdown(f'Successfully updated the alert: "{alert_id}"', result, removeNull=True)
+    outputs = {
+        'AlertID': alert_id,
+        'State': result.get('state'),
+        'Remediation': result.get('remediation'),
+        'LastUpdateTime': result.get('last_update_time'),
+        'Comment': result.get('comment'),
+        'ChangedBy': result.get('changed_by')
+    }
+
+    results = CommandResults(
+        outputs_prefix='CarbonBlackEEDR.Alert',
+        outputs_key_field='AlertID',
+        outputs=outputs,
+        readable_output=readable_output,
+        raw_response=result
+    )
+    return results
+
+
+def list_devices_command(client: Client, args: Dict) -> Union[CommandResults, str]:
+    device_id = argToList(args.get('device_id'))
+    status = argToList(args.get('status'))
+    device_os = argToList(args.get('device_os'))
+    last_contact_time = {
+        'start': args.get('start_time'),
+        'end': args.get('end_time')
+    }
+    if args.get('start_time') and not args.get('end_time'):
+        last_contact_time = {
+            'start': args.get('start_time'),
+            'end': datetime.now().strftime('%Y-%m-%dT%H:%M:%S.000Z')
+        }
+    ad_group_id = argToList(args.get('ad_group_id'))
+    policy_id = argToList(args.get('policy_id'))
+    target_priority = argToList(args.get('target_priority'))
+    limit = args.get('limit')
+    sort_field = args.get('sort_field', '')
+    sort_order = args.get('sort_order')
+    contents = []
+
+    result = client.devices_list_request(device_id, status, device_os, last_contact_time, ad_group_id, policy_id,
+                                         target_priority, limit, sort_field, sort_order)
+    devices = result.get('results', [])
+    if not devices:
+        return 'No devices were found.'
+    for device in devices:
+        contents.append({
+            'ID': device.get('id'),
+            'Name': device.get('name'),
+            'OS': device.get('os'),
+            'LastInternalIpAddress': device.get('last_internal_ip_address'),
+            'LastExternalIpAddress': device.get('last_external_ip_address'),
+            'LastContactTime': device.get('last_contact_time'),
+            'LastLocation': device.get('last_location'),
+            'PolicyName': device.get('policy_name'),
+            'Quarantined': device.get('quarantined'),
+            'status': device.get('status'),
+            'TargetPriority': device.get('target_priority')
+        })
+
+    # endpoint = Common.Endpoint(
```

Comment:
why is this commented out?

Reply:
Needed to run the command in demisto. Will undo